src/components/MerchantsPage/gam/NoCampaignsBox.tsx

```diff
@@ -5,25 +5,40 @@ import {
 } from '../../../utilities/general/responsive';
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
+import {
+  useModalPaymentDispatch,
+  ModalPaymentConstants,
+  ModalPaymentTypes,
+} from '../../../utilities/hooks/ModalPaymentContext';
+import Modal from '../../ModalPayment';
 
 const NoActiveCampaignsBox = () => {
   const { t } = useTranslation();
 
+  const ModalPaymentDispatcher = useModalPaymentDispatch(null);
+
+  const openModal = (event) => {
+    event.preventDefault();
+    ModalPaymentDispatcher({
+      type: ModalPaymentConstants.SET_MODAL_VIEW,
+      payload: ModalPaymentTypes.modalPages.donation,
+    });
+  };
+
   return (
     <NoCampaignBox>
       <TextContainer>
         <Heading>{t('gamHome.noCampaignsBox.CTA')}</Heading>
         <SubHeading>{t('gamHome.noCampaignsBox.description')}</SubHeading>
       </TextContainer>
-      <Button
-        className="button--filled"
-        onClick={(e) => {
-          e.preventDefault();
-          window.open('https://www.gofundme.com/f/gift-a-meal', '_blank');
-        }}
-      >
+      <Button className="button--filled" onClick={openModal}>
         {t('gamHome.giftButton')}
       </Button>
+      <Modal
+        sellerId="send-chinatown-love"
```

Comment:
I don't think we want the pooled donation to go to "send-chinatown-love" we want for it to go to something like apex for youth's project id: https://docs.google.com/document/d/12ViA_fIQ060OvvnzJDyBM88SoTGpdcuD0rDk_veNjoc/edit